apps/www/app/_components/tokens-comp/semantic/semantic-table.tsx

```diff
@@ -0,0 +1,88 @@
+import { Heading, Table } from '@digdir/designsystemet-react';
+import { groupBy } from 'ramda';
+import type { HTMLAttributes } from 'react';
+import { capitalizeString } from '~/_utils/string-helpers';
+import type { PreviewToken } from '../types';
+import {
+  BorderRadius,
+  ComputedValue,
+  Opacity,
+  Shadow,
+  Size,
+} from './seamantic-previews';
+
+type TokenTableProps = {
+  tokens: PreviewToken[];
+} & HTMLAttributes<HTMLDivElement>;
+
+const groupedByPathIndex = (index = 0) =>
+  groupBy((token: PreviewToken) => token.path[index] || 'rest');
+
+const getValuePreview = (variable: string, value: string) => {
+  if (/^--ds-size.*\d$/.test(variable)) {
+    return <Size value={value} />;
+  }
+  if (/^--ds-border-radius(?!.*(scale|base)$)/.test(variable)) {
+    return <BorderRadius value={value} />;
+  }
+
+  if (/^--ds-shadow/.test(variable)) {
+    return <Shadow value={value} />;
+  }
+
+  if (/^--ds-opacity/.test(variable)) {
+    return <Opacity value={value} />;
+  }
+
+  return <code>{value}</code>;
+};
+
+const getValueRender = (variable: string, value: string) => {
+  if (!/opacity|shadow/.test(variable)) {
+    return <ComputedValue value={value} />;
+  }
+
+  return <code>{value}</code>;
```

Comment:
Should this be a button you can click to copy?

Reply:
Was thinking we could add this later? I can also just change it to use the `ClipboardButton` 🤔

Reply:
If the clipboard looks OK, then I say go for it

Reply:
Didn't look ok, so saving this for later.